rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -288,6 +305,7 @@ private LanguageClient availableClient() {
     @Override
     public void connect(LanguageClient client) {
         this.client = client;
+        this.dynamicCapabilities = new DynamicCapabilities(client, exec, List.of(new CompletionCapability()));
```

Comment:
shouldn't we look if somehow we already received some registrations?

Reply:
ah yes, connect really should have happened first, as the registerLanguage only go over an initialized lsp connection. okay, ignore my comment.